helm-charts/konk/scripts/provision.sh

```diff
@@ -40,6 +48,13 @@ then
     --key=/etc/kubernetes/pki/ca.key
   kubectl -n $NAMESPACE label secret $FULLNAME-ca $LABELS
 fi
+if ! kubectl -n $NAMESPACE get secret $FULLNAME-etcd-ca
```

Comment:
we need to check for 'not found' errors specifically. We could run into a scenario where a 503 service unavailable causes the secret to get overridden

Reply:
Makes sense. That would also be the case for the k8s CA as well, right?

Reply:
This logic might work for you. Experiment with it...
```suggestion
if [ "$(kubectl -n $NAMESPACE get secret $FULLNAME-etcd-ca --ignore-not-found 2>&1 )" = "" ]
```
It checks that the response is successful, but empty, which is what you would expect from this command when a secret does not exist.